reaction-contrib/meteor-notifications-sms
# #1 Update call to formatPhoneNumber to await for result

## Issue
`formatPhoneNumber` was updated in core to be an async function so that a library it depends on - `libphonenumber-js` is loaded dynamically. 

## Solution
Update this plugins call to formatPhoneNumber to await for result.

## Testing
1) Use the React branch from this PR: https://github.com/reactioncommerce/reaction/pull/4517
2) Add an address under your profile that includes a phone number without the country calling code. Example: 5555555555 (no 1 at beginning)
3) Dashboard->SMS Notifications - Place some dummy values and save
4) Add logging to imports/plugins/custom/meteor-notifications-sms/server/methods/sms.js, after line 95: `console.log("formattedPhone:", formattedPhone);`
5) Client console. Run: `Meteor.call("sms/send", "test", Meteor.userId(), ReactionCore.getPrimaryShopId());`
6) Confirm server console reads: `formattedPhone: 15555555555` (or correct calling code of selected country).
